packages/legend-query/src/stores/QueryBuilderSetupState.ts

```diff
@@ -31,12 +30,17 @@ import {
   getMilestoneTemporalStereotype,
   PackageableElementExplicitReference,
   RuntimePointer,
+  DEFAULT_BUSINESS_DATE_MILESTONING_PARAMETER_NAME,
+  DEFAULT_PROCESSING_DATE_MILESTONING_PARAMETER_NAME,
+  MILESTONING_STEROTYPE,
+  observe_ValueSpecification,
 } from '@finos/legend-graph';
 
 export class QueryBuilderSetupState {
   queryBuilderState: QueryBuilderState;
   _class?: Class | undefined;
-  classMilestoningTemporalValues: ValueSpecification[] = [];
+  _businessDate: ValueSpecification | undefined;
```

Comment:
Let's move these to the bottom of the list of attributes and put a `TODO` to indicate we're working towards changing this